nixui/options/attribute.py

```diff
@@ -37,7 +38,9 @@ def __lt__(self, other):
 
     def __str__(self):
         return '.'.join([
-            f'"{attribute}"' if '.' in attribute else attribute
+            attribute
+            if re.match(r'^[a-zA-Z\_][a-zA-Z0-9\_\'\-]*$', attribute)
```

Comment:
I'm not sure if this regex should include `\'`.

Reply:
Hmm, it's based on https://github.com/NixOS/nix/blob/master/src/libexpr/lexer.l#L97

Reply:
Thanks. I think it's worth it to add a comment here, referencing either your link or https://github.com/NixOS/nixpkgs/blob/8da27ef161e8bd0403c8f9ae030ef1e91cb6c475/pkgs/tools/nix/nixos-option/libnix-copy-paste.cc#L52

Reply:
sounds good

resolved with https://github.com/nix-gui/nix-gui/pull/80/commits/82b5b451cc3b3d6a1d1d73553785978a8129b5cb